### PR TITLE
Automaticaly deploy to pypi.org

### PR DESCRIPTION
**Pull request by ClementGautier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,18 @@ CLI and SDK for interacting with Substra platform.
...
+  - [Install](#install)
+  - [Usage](#usage)
+    - [CLI](#cli)
+    - [SDK](#sdk)
+  - [Documentation](#documentation)
+  - [Examples](#examples)
+  - [Contributing](#contributing)
+    - [Setup](#setup)
+    - [Documentation](#documentation-1)
+    - [Deploy](#deploy)
```
Comment (samlesu) [inline README.md]: This change seems not related to the PR scope and I'm not sure it's good idea to:
- nest all items inside `Substra`
- add the table of contents
- expand the section `Usage` and `Contributing`

File: LICENCE [hunk 2]
```diff
@@ -1,190 +0,0 @@
-                                 Apache License
```
Comment (samlesu) [inline LICENCE]: @inalgnu can you confirm that we can safely remove this LICENCE?

Comment (inalgnu) [reply LICENCE]: yes! 
My bad, it was a wrong file name

File: README.md (hunk 1 above)
Comment (ClementGautier) [reply README.md]: This change was automatically performed by my IDE to be honest. I pushed it because it updated a table of content that existed but was incomplete. I will undo the change so it stay simple

**Comment by Kelvin-M** [comment]
Nice to note that we use an encrypted token as suggested  in https://docs.travis-ci.com/user/deployment/pypi/
Thank you Clément !
